projects/dsp-ui/src/lib/viewer/values/uri-value/uri-value.component.html

```diff
@@ -1,5 +1,7 @@
 <div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
-    <p class="rm-value">{{valueFormControl.value}}</p>
+    <a target="_blank" href="{{valueFormControl.value}}">
```

Comment:
Could the `<p>` element just contain the `<a>` tag?

https://stackoverflow.com/questions/13911274/can-a-p-tag-in-a-tag#:~:text=elements%20may%20not%20contain,This%20is%20not%20allowed.

Reply:
fixed in d5350ec

Reply:
thanks